doc/source/tutorial/stats/continuous_ksone.rst

```diff
@@ -4,4 +4,39 @@
 KSone Distribution
 ==================
 
+
+This is the distribution of maximum positive differences between an
+empirical distribution function, computed from n samples or observations,
+and a comparison (or target) cumulative distribution function.
+
+Writing :math:`D_n^+ = \sup_t \left(F_{target}(t)-F_{empirical,n}(t)\right)`,
+KSone is the distribution of the :math:`D_n^+` values.
+(The distribution of :math:`D_n^- = -\inf_t \left(F_{target}(t)-F_{empirical,n}(t)\right)`
+ differences follows the same distribution, so ksone can be used for one-sided tests on either side.)
+
+
+One shape parameter
```

Comment:
Not really I think. Most of these have been converted to Sphinx from LaTeX a long time ago, that's why the format looks fairly uniform. But I'm not sure I can point to a template or one function as the preferred one.